modules/ob-example-configuration-for-defining-parameter-values.adoc

```diff
@@ -0,0 +1,166 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="example-configuration-for-defining-parameter-values_{context}"]
+= Example configuration for defining parameter values
+
+The following examples show how to define parameters in a build strategy and assign values to those parameters by using a `Build` CR. You can also assign values to the `build-args` parameter, which can be a config map, secret, or hard-coded value.
+
+.Example: Defining parameters in a `ClusterBuildStrategy` CR
+The following example shows a `ClusterBuildStrategy` CR that define several parameters:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: ClusterBuildStrategy
+metadata:
+  name: buildkit
+  ...
+spec:
+  parameters:
+  - name: build-args
+    description: "The ARG values in the format KEY=VALUE."
+    type: array
+    defaults: []
+  - name: cache
+    description: "Configure BuildKit's cache usage."
+    type: string
+    default: registry
+  ...
+  steps:
+  ...
+----
+
+.Example: Assigning values to parameters in a `Build` CR
+The above `ClusterBuildStrategy` CR defines a `cache` parameter and you can specify the value of the `cache` parameter in your `Build` CR, as shown in the following example:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: a-build
+  namespace: a-namespace
+spec:
+  paramValues:
+  - name: cache
+    value: disabled
+  strategy:
+    name: buildkit
+    kind: ClusterBuildStrategy
+  source:
+  ...
+  output:
+  ...
+----
+
+.Example: Creating a `ConfigMap` CR to control a parameter centrally
+If you want to use the `cache` parameter for multiple builds and control its usage centrally, then you can create a `ConfigMap` CR, as shown in the following example:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
```

Comment:
```suggestion
apiVersion: v1
```
The configmap refers to an API that's independent of shipwright.io, hence, that stays `v1`.